made the ui change for translate button for smaller devices

kripa-sindhu-007 opened the pull request:
Moved the Translate to English/Hindi button to the main page on smaller devices 

## Motivation and Context
1. An issue was raised for this in the repo
2. Issue-109[](https://github.com/rbjks/rbjks.github.io/issues/109)


## Screenshots (if appropriate):
![Screenshot 2024-06-13 170753](https://github.com/rbjks/rbjks.github.io/assets/152695457/11ac5ca3-5a5a-4c70-ba44-c5b9b9835bb9)


## Types of Changes
Just made a new div for that button and a few lines of CSS

## I have only changed lines in index file and its CSS. but a lot of changes are seen. One of the reasons I can think of is the use of Preetier for formatting docs. I would urge to the maintainer of this repo to make sure that I have not made any necessary changes

adityaraute reviewed (changes requested):
Hi, Your changes on the main page look good.

There are two issues in your PR that we'd like to get fixed:

1. The button only appears on the home index page but not on Contact or Activities page. Please add the button there, too.
2. Your PR has made changes to 14 files, most of them linting changes. Please get rid of those unnecessary changes for us to merge the PR.

Thanks!

kripa-sindhu-007 commented:
I have added that button to all the pages.
For your second concern as I mentioned I haven't change anything I tried to see what changes are made somewhere its change of Inverted commas from " " to ' '.  Somewhere some blank lines are added . I have formatted the document using Preetier plugin in Vs Code that's why these changes are visible. 

P.S: I added a start command in package,json to deploy the webpage locally which was not present earlier
